bokeh/util/serialization.py

```diff
@@ -569,6 +570,18 @@ def decode_base64_dict(data):
         array = array.reshape(data['shape'])
     return array
 
+def encode_base64_string(str, value):
+    ''' Encode a ascii string using Base64.
+
+    Args:
+        str : a string to encode
```

Comment:
arg name is `value` (in general, should try to avoid using built-in Python names like `str` as variable names)